### DISCUSSION TITLE
Paste command output to message box

### DISCUSSION THREAD

**Discussion by Emilgardis** [Chatterino/chatterino2#3773] [Ideas]
Similar to FFZ option "Paste this message into chat rather than sending it directly. "

<img width="758" alt="image" src="https://user-images.githubusercontent.com/1502855/170842639-d5224260-1117-4603-a078-b3ba31fe1bdc.png">

and chatty command `/insert`

<img width="688" alt="image" src="https://user-images.githubusercontent.com/1502855/170843342-3589fe50-4303-4e23-ad24-826459a94678.png">

Chatterino should provide a command `/insert` (bikeshed needed) that when used in a command like the following

<img width="330" alt="image" src="https://user-images.githubusercontent.com/1502855/170842698-644e8e76-85c7-4a53-9726-91a6cf9e412e.png">

Put the message in the chatbox in the same split.

It should work in moderation buttons and commands

**Comment by KararTY** [comment]
There's a workaround way to get this functionality using the /copy command recently implemented in https://github.com/Chatterino/chatterino2/pull/3763

You'd just have to paste the result back in.

Example command:
```
/copy OCEAN MAN 🌊 😍 Take me by the hand ✋ lead me to the land that you understand 🙌 🌊 OCEAN MAN 🌊 😍 The voyage 🚲 to the corner of the 🌎 globe is a real trip 👌 🌊 OCEAN MAN 🌊 😍 The crust of a tan man 👳 imbibed by the sand 👍 Soaking up the 💦 thirst of the land 💯
```

**Reply by Emilgardis** [reply]
Yeah I saw that while trying to implement this myself, this works for now, but not very convenient :D
